perception_dataset/rosbag2/autoware_msgs.py

```diff
@@ -44,9 +44,15 @@ def object_classification_to_category_name(object_classification) -> str:
         2: "truck",
         3: "bus",
         4: "trailer",
-        5: "motorcycle",
+        5: "motorbike",
```

Comment:
please keep to use `motorcycle` following a real dataset 🙇

Reply:
Sure! Thank you for spotting 🙇🏼

Reply:
fixed in https://github.com/tier4/tier4_perception_dataset/pull/28/commits/fdd6db2e8a18f6d91b257670ae36a3bc50a1d253